src/components/daily/bolustooltip/BolusTooltip.js

```diff
@@ -308,7 +303,7 @@ class BolusTooltip extends React.Component {
     );
     const inputLine = inputTime && (
       <div className={styles.input}>
-        <div className={styles.label}>{t('Entered at {{date}}', {date: this.getFormatedInputTime(bolus, timePrefs)})}</div>
+        <div className={styles.label}>{t('Entered at {{date}}', {date: formatInputTime(bolus.inputTime, timePrefs)})}</div>
```

Comment:
`'bolus.inputTime' is missing in props validationeslintreact/prop-types`
Add line 437: `inputTime: PropTypes.string,`
```
  bolus: PropTypes.shape({
    type: PropTypes.string.isRequired,
    inputTime: PropTypes.string,
  }).isRequired,
```